### PR TITLE
RavenDB-9138 & RavenDB-7936

### PR DESCRIPTION
**Pull request by grisha-kotler**
http://issues.hibernatingrhinos.com/issue/RavenDB-9138
http://issues.hibernatingrhinos.com/issue/RavenDB-7936

### CODE REVIEW

File: src/Raven.Studio/typescript/viewmodels/database/tasks/backupNow.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        this.isFullBackup(true);
+        this.confirm();
+    }
+
+    incrementalBackup() {
+        this.isFullBackup(false);
+        this.confirm();
+    }
+
+    protected getCofirmButton(): HTMLElement {
```
Comment (ml054) [inline src/Raven.Studio/typescript/viewmodels/database/tasks/backupNow.ts]: Is it used? + it has typo.

Comment (grisha-kotler) [reply src/Raven.Studio/typescript/viewmodels/database/tasks/backupNow.ts]: removed

File: src/Raven.Server/Documents/PeriodicBackup/PeriodicBackupRunner.cs [hunk 2]
```diff
@@ -963,51 +942,86 @@ public void RemoveInactiveCompletedTasks()
...
-                Configuration = newConfiguration
-            };
+                // backup frequency hasn't changed
+                // and we have a scheduled backup
+                return;
+            }
+
+            existingBackupState.UpdateTimer(GetTimer(newConfiguration, backupStatus));
+
+            //if (taskState == TaskStatus.ActiveByCurrentNode &&
```
Comment (ayende) [inline src/Raven.Server/Documents/PeriodicBackup/PeriodicBackupRunner.cs:993]: Why so much commented code?

Comment (grisha-kotler) [reply src/Raven.Server/Documents/PeriodicBackup/PeriodicBackupRunner.cs:993]: i'll remove this

**Pull request merged**